lib/osm.cc

```diff
@@ -81,7 +81,7 @@ BOOST_PYTHON_MODULE(_osm)
     ;
     class_<osmium::Box>("Box",
         "A bounding box around a geographic area. Such a box consists of two "
-        ":py:class:`osmium.osm.Location`s. Those locations may be invalid in "
+        ":py:class:`osmium.osm.Location`'s. Those locations may be invalid in "
```

Comment:
That unfortunately makes the plural s a genitive s. I couldn't find any hint in the documentation how to add a plural s, so the better way forward is to reformulate the entire sentence.